src/pe_reports/pages.py

```diff
@@ -0,0 +1,150 @@
+"""Collect and distribute graphical data to readable charts in the presentation."""
+
+# Third-Party Libraries
+import pandas as pd
+from pptx.chart.data import CategoryChartData
+from pptx.enum.chart import XL_CHART_TYPE
+from pptx.util import Inches
+from stylesheet import Graph, Paragraph
+
+# Use the following function to locate "Text Box" element in slide.
+# find_shape = Paragraph.shapes_find(prs, slide)
+
+
+class Pages:
+    """Class containing the presentation page information."""
+
+    def cover(prs):
+        """Page 1: Cover page of presentation."""
+        slide = prs.slides[0]
+        shape = Paragraph.shapes(prs, slide)
+        text_frame = Paragraph.text_frame(prs, shape)
+        frame = text_frame.paragraphs[0]
+        run = frame.add_run()
+        df_customer = pd.read_csv("src/pe_reports/data/csv/dhs_customer.csv")
+        dates = (
+            df_customer.iloc[0]["start_date"] + " to " + df_customer.iloc[0]["end_date"]
+        )
+        run.text = (
+            "Prepared for: "
+            + str(df_customer.iloc[0]["name"])
+            + "\nReporting period: "
+            + str(dates)
+        )
+        font = run.font
+        font = Paragraph.text_style_title(prs, font)
+        return prs
+
+    def overview(prs):
+        """Page 2: Posture & Exposure Report Overview."""
+        slide = prs.slides[1]
+        shape = Paragraph.shapes(prs, slide)
+
+        # Overeview Value - Credentials exposed in recent posts
```

Comment:
I'm pretty sure this is a typo.  If so, note there are several other occurrences of it below that should be fixed as well.
```suggestion
        # Overview Value - Credentials exposed in recent posts
```

Reply:
Ran Code Spell check and updated where appropriate. 

Resolved in 8806233